ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -118,6 +126,79 @@ tests:
     test:
     - ref: windows-e2e-operator-test-with-custom-vxlan-port
     workflow: openshift-e2e-vsphere-ccm-ovn-hybrid-install
+- always_run: false
+  as: vsphere-e2e-upgrade
+  steps:
+    cluster_profile: vsphere
+    test:
+    - as: install
+      cli: latest
+      commands: |
+        export WMCO_NS=openshift-windows-machine-config-operator
+        oc create namespace $WMCO_NS
+        oc label --overwrite=true ns $WMCO_NS openshift.io/cluster-monitoring=true \
+        pod-security.kubernetes.io/enforce=privileged
+        # Deploy the bundle which will fail to come up due to it using the operator image via a tag that is not able to
+        # be made public. The csv needs to be patched to point the image via the tag that is pullable by the job.
+        operator-sdk run bundle --timeout=10m --security-context-config restricted -n $WMCO_NS "$PREVIOUS_BUNDLE" \
```

Comment:
will this hit the 10 min timeout every time?

Reply:
Can't you patch the CSV first?

Reply:
The timeout will not be hit as there is an issue in operator-sdk which will not respect the timeout parameter correctly.  https://github.com/operator-framework/operator-sdk/pull/6440